### DISCUSSION TITLE
VS Code + Copilot (Linux) RAM usage Spikes + lockups

### DISCUSSION THREAD

**Discussion by Free-Radical** [community/community#47801] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Vs code/Copilot under Linux Mint 21/Plasma intermittently gets very busy leading to lockup at which Ram usage goes from ~2 GB to 8-9 GB's. Machine locksup and reports VS code "not responding".

Happens with only 1 small python using streamlit module open in editor.

Some times machine comes back to normal other times I have to kill/restart VS Code.

Happening a lot , 5 times a day

OTherwise system is always stable.

STEPS TO REPRODUCE-
1. Load latest VS CODE on to latest Linux Mint 21 w/ Plasma
2. ensure Python has streamlit installed 
3. use basic streamlit hello world program

VS CODE INFO-
Version: 1.75.1
Commit: 441438abd1ac652551dbe4d408dfcec8a499b8bf
Date: 2023-02-08T21:35:30.018Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Linux x64 5.19.0-28-generic
Sandboxed: No

SYS Info-

CPUs | AMD Ryzen 5 5600U with Radeon Graphics (12 x 4043)
-- | --
GPU Status | 2d_canvas: enabled canvas_oop_rasterization: disabled_off direct_rendering_display_compositor: disabled_off_ok gpu_compositing: enabled multiple_raster_threads: enabled_on opengl: enabled_on rasterization: enabled raw_draw: disabled_off_ok skia_renderer: enabled_on video_decode: disabled_software video_encode: disabled_software vulkan: disabled_off webgl: enabled webgl2: enabled webgpu: disabled_off
Load (avg) | 1, 1, 1
Memory (System) | 13.51GB (4.68GB free)
Process Argv | --unity-launch
Screen Reader | no
VM | 0%

**Comment by thomfilg** [comment]
I have the same issue here..

**Reply by Free-Radical** [reply]
well the important part is that Microsoft a $2.15 trillion doesn't give a $hit!

**Reply by nikolasstow** [reply]
A year on and I'm having the same issue.

**Reply by marcoveio-diadema** [reply]
having a very hard time too!

**Reply by danielwu97** [reply]
opening one project with pycharm. pycharm used 1.6G. copilot-language-server used 1.9G.
what a huge huge.
